solvers/ADMM.py

```diff
@@ -0,0 +1,80 @@
+from benchopt import BaseSolver
+from benchopt.stopping_criterion import SufficientDescentCriterion
+from benchopt import safe_import_context
```

Comment:
you add this new solver on the laplace PR on purpose?

Reply:
It is an issue of rebasing